vulture_allowlist.py

```diff
@@ -119,40 +110,15 @@
 
 MonkeyExploitation.label
 
-# Unused plugin infrastructure
-HTTPIslandAPIClient.get_agent_plugin
-HTTPIslandAPIClient.get_agent_configuration_schema
-
-# Remove after #2518
-AgentHeartbeat
-
 Lock.exc_type
 Lock.exc_val
 Lock.exc_tb
 Lock.blocking
 Lock.locked
 
-# Remove after we construct it
-QueuedAgentEventPublisher
-
-# Remove after #2638
-AgentPlugin.plugin_manifest
-AgentPlugin.config_schema
-AgentPlugin.source_archive
-
-AgentPluginManifest
-AgentPluginManifest.supported_operating_systems
-AgentPluginManifest.description
-AgentPluginManifest.name
-AgentPluginManifest.plugin_type
+# Pydantic configurations are not picked up
 AgentPluginManifest.title
+AgentPluginManifest.description
 AgentPluginManifest.link_to_documentation
 AgentPluginManifest.safe
-
-IAgentPluginRepository
-
-# Remove after #2710
-IAgentPluginRepository.get_plugin_catalog
-
-# Remove once #2783 is done
-PluginConfigurationValidationError
+AgentPluginManifest.supported_operating_systems
```

Comment:
Why have two sections for "Pydantic configurations are not picked up"? Just add these to the one above.